src/api/authorityApi.ts

```diff
@@ -0,0 +1,80 @@
+import Axios from 'axios';
+import { Dispatch } from 'redux';
+
+import { addNotification } from '../redux/actions/notificationActions';
+import i18n from '../translations/i18n';
+import { Authority } from '../types/authority.types';
+import { StatusCode, USE_MOCK_DATA } from '../utils/constants';
+import { getIdToken } from './userApi';
+
+export const getAuthorities = async (name: string, dispatch: Dispatch) => {
+  const url = '/authority';
+
+  // remove when Authorization headers are set for all requests
+  const idToken = USE_MOCK_DATA ? '' : await getIdToken();
+  const headers = {
+    Authorization: `Bearer ${idToken}`,
+  };
+
+  try {
+    const response = await Axios.post(url, { name }, { headers });
+
+    if (response.status === StatusCode.OK) {
+      return response.data;
+    } else {
+      dispatch(addNotification(i18n.t('feedback:error.get_authorities'), 'error'));
+    }
+  } catch {
+    dispatch(addNotification(i18n.t('feedback:error.get_authorities'), 'error'));
+  }
+};
+
+export const getAuthorityByFeideId = async (feideId: string, dispatch: Dispatch) => {
+  const url = '/authority';
+
+  // remove when Authorization headers are set for all requests
+  const idToken = USE_MOCK_DATA ? '' : await getIdToken();
+  const headers = {
+    Authorization: `Bearer ${idToken}`,
+  };
+
+  try {
+    const response = await Axios.post(url, { feideId }, { headers });
```

Comment:
jeg tenker at vi skulle sendt en GET til backend, så kunne man der håndtert ulogiske http-kall bak kulissene. men det viktigste nå er vel egentlig bare å få på plass noe, så det får bare ligge(?)

Reply:
skjønner ikke helt hva du mener her.

Reply:
1. Vi sender GET authority til vår backend
2. Backend sender sært POST-kall til APIet
3. Vi får svar fra vårt GET-kall til backend

Reply:
greia er at vi kan sende enten feideId eller orcId til samme endepunkt for å GET'e en autoritet. synes det er greit slik det er foreløpig